app/examples/qutebrowser

```diff
@@ -0,0 +1,111 @@
+" Options
+set adblocker=update
+set devtoolsposition=split
+set downloadmethod=confirm
+set firefoxmode=always
+set fontsize=16
+set norestoretabs
+set permissioncamera=ask
+set permissiondisplaycapture=ask
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set tabclosefocusright
+set tabreopenposition=next
+set windowtitle=full
+set mapsuggestposition=bottomleft
+
+" Mappings
+nmap <Up> <action.scrollUp>
+nmap <Down> <action.scrollDown>
+nmap <Left> <action.scrollLeft>
+nmap <Right> <action.scrollRight>
+nmap <PageUp> <action.scrollPageUp>
+nmap <PageDown> <action.scrollPageDown>
+nmap <Home> <action.scrollTop>
+nmap <End> <action.scrollBottom>
+
+nmap o <action.toExploreMode>
+nmap wi <:devtools>
+nmap <C-PageUp> <action.previousTab>
+nmap <C-PageDown> <action.nextTab>
+nmap <F5> <action.reload>
+nmap <C-F5> <action.reloadWithoutCache>
+nmap Sh <:history>
+nmap Ss <:help>
+nmap <C-A-p> <:print>
+nmap <C-q> <:quit>
+nmap <C-W> <:quit>
+nmap ZQ <:quit>
+nmap gd <:write>
+nmap <C-Tab> <action.nextTab>
+nmap <C-S-PageUp> <action.moveTabBackward>
+nmap <C-S-PageDown> <action.moveTabForward>
+nmap <C-S-Tab> <action.previousTab>
```

Comment:
First you set it here, then you set it to `nop` below, can't we just leave both of these lines out? This mappings isn't mapped by default anyway.

Reply:
Good point. Confusingly, qutebrowser sets these both by default.

Reply:
Actually, now there is a mapping for going to the next tab with `<C-Tab>`, but you can't go back with `<C-S-Tab>`, I was mistaken before, we should just have removed the nop line and leave this one as is I think.

Reply:
I've readded the mapping.